src/app/phase-tester-response/issue-responded/issue-responded.component.ts

```diff
@@ -0,0 +1,27 @@
+import {Component, OnInit, ViewChild} from '@angular/core';
+import {ACTION_BUTTONS, IssueTablesComponent} from '../../shared/issue-tables/issue-tables.component';
+import {Issue, STATUS} from '../../core/models/issue.model';
+
+@Component({
+  selector: 'app-issue-responded',
+  templateUrl: './issue-responded.component.html',
+  styleUrls: ['./issue-responded.component.css']
+})
+export class IssueRespondedComponent implements OnInit {
+
+  @ViewChild(IssueTablesComponent) table: IssueTablesComponent;
+
+  readonly displayedColumns: string[] = ['id', 'title', 'type', 'severity', 'actions'];
```

Comment:
@KevinCJH Do you think we should make the displayed columns a set of `enums` as well? That way future developers know of the existing column names especially when the IDE will prompt them.

Reply:
Yes, I think that will be good